services/ansibler/server/domain/usecases/update_no_proxy_envs.go

```diff
@@ -134,7 +134,8 @@ func createNoProxyList(desiredNodePools []*spec.NodePool, desiredLbs []*spec.LBc
 	}
 
 	// if "svc" isn't in noProxyList the admission webhooks will fail, because they will be routed to proxy
-	noProxyList = fmt.Sprintf("%s,%s,", noProxyList, "svc")
+	// "metadata,metadata.google.internal,169.254.169.254,metadata.google.internal." are required for GCP VMs
```

Comment:
Is the IP `169.254.169.254` needed there or does it not work without the IP?

Reply:
It is needed. At least according to https://cloud.google.com/compute/docs/troubleshooting/troubleshoot-metadata-server#failed-proxy

"...If the VM is behind a proxy, you must set the NO_PROXY configuration for both the IP address and Hostname."

But TBH I haven't tried to build the k8s cluster without the IP.